go.mod

```diff
@@ -163,6 +162,7 @@ require (
 	github.com/huaweicloud/huaweicloud-sdk-go-v3 v0.0.87
 	github.com/labd/commercetools-go-sdk v0.3.2
 	github.com/nacos-group/nacos-sdk-go/v2 v2.0.1
+	github.com/rabbitmq/amqp091-go v1.3.4
```

Comment:
`github.com/rabbitmq/amqp091-go` is the new version of `github.com/streadway/amqp` and it's maintained by RabbitMQ now